libpay-braintree/src/main/scala/com/wix/pay/braintree/BraintreeMerchantParser.scala

```diff
@@ -1,6 +1,6 @@
 package com.wix.pay.braintree
 
 trait BraintreeMerchantParser {

```

Comment:
Hmm.... Remember what I said when you talked to me about this PR? That I believe one parser should handle both cases? This just maybe the way to accomplish that.
Would you consider to re-think it?